try {
    console.log(main);
    const name  =  'nieem';
} catch (error) {
    console.log('error');
}

console.log('500000000000000');